test/Microsoft.AspNetCore.Blazor.Razor.Extensions.Test/HtmlBlockPassTest.cs

```diff
@@ -75,7 +145,7 @@ public void Execute_RewritesHtml_CSharpInAttributes()
   </head>
 </html>");
 
-            var expected = NormalizeContent(@"<div>foo</div>");
+            var expected = NormalizeContent("<div>foo</div>\n  ");
```

Comment:
These tests changed because the newline that follows the element has been merged in.